cinema/views.py

```diff
@@ -35,7 +36,7 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
+    authentication_classes = (JWTAuthentication,)
```

Comment:
You set JWTAuthentication as default authentication class in your settings, so you can delete this in all places.